pkgs/applications/radio/yaac/default.nix

```
@@ -0,0 +1,34 @@
+{ makeWrapper, lib, stdenv, fetchurl, unzip, jre8 }:
+
+stdenv.mkDerivation rec {
+  version = "1.0-beta181";
+  pname = "YAAC";
+
+  src = fetchurl {
+    url = "https://www.ka2ddo.org/ka2ddo/YAAC.zip";
+    sha256 = "564071bb51fb4a91850ff53ff7d0c9477c1715d7b11eed9e97c89ead9df7332b";
+  };
+
+  nativeBuildInputs = [ unzip makeWrapper ];
+
+  unpackPhase = "unzip $src -d $out";
+
+  # TODO: Make Desktop Icon
+
+  installPhase = ''
+    mkdir -p $out/bin
+    makeWrapper ${jre8}/bin/java $out/bin/yaac \
+      --add-flags "-jar $out/YAAC.jar" \
+      --set _JAVA_OPTIONS '-Dawt.useSystemAAFontSettings=on' \
+      --set _JAVA_AWT_WM_NONREPARENTING 1
+  '';
+
+  meta = with lib; {
+    description = "Yet Another APRS Client";
+    homepage = "https://www.ka2ddo.org/ka2ddo/YAAC.html";
+    license = licenses.lgpl3Only;
+    platforms = platforms.linux;
```

```suggestion
    platforms = platforms.unix;
```
It seems to work in any environment where java runs, at least on unix.

---

Updated in latest.